src/reducers/application.js

```diff
@@ -126,18 +126,22 @@ export default (state = initialState, action) => {
       };
     }
     case CONTENT_TYPES_LOADED: {
+      const { data } = action.payload.contentTypes;
       return {
         ...state,
-        contentTypes: action.payload.contentTypes.data.reduce(
-          (accumulator, contentType) => ({
-            ...accumulator,
-            [contentType.attributes.type]: {
-              name: contentType.attributes.name,
-              description: contentType.attributes.description,
-            },
-          }),
-          {},
-        ),
+        contentTypes:
+          data instanceof Array
+            ? data.reduce(
+                (accumulator, contentType) => ({
+                  ...accumulator,
+                  [contentType.attributes.type]: {
+                    name: contentType.attributes.name,
+                    description: contentType.attributes.description,
+                  },
+                }),
+                {},
+              )
+            : data,
```

Comment:
let's return an empty object instead of `data` as the fallback here

Reply:
@justafish I think what will happen, _although I haven't tested it_, is that if you only have one Content Type, instead of an `array`, its just an object.

Reply:
Which is an especially frustrating part of the API.